### PR TITLE
Add team section

### PR DESCRIPTION
**Pull request by vipulgupta2048**
The pull request is not complete and some more edits are required on the volunteer list. 

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: content/team.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Abhishek Upperwal
+- Akshay Gaikwad
+- Anirudh Menon
+- Aravind Sekar
+- Arun
+- Bharat
+- Bhavin Gandhi
+- Chandan Kumar
+- Darshna Das
+- Divya
```
Comment (bhavin192) [inline content/team.md:20]:
```suggestion
- Divya Goswami
```

File: content/team.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Duranjai Malik
+- Farhaan B
```
Comment (bhavin192) [inline content/team.md:22]:
```suggestion
- Farhaan Bukhsh
```

File: content/team.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Gajendra
```
Comment (bhavin192) [inline content/team.md:23]:
```suggestion
- Gajendra Deshpande
```

File: content/team.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- Garvit Kulshreshtha
+- Girish
```
Comment (bhavin192) [inline content/team.md:25]:
```suggestion
- Girish Joshi
```

File: content/team.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- Jatin Goel
+- Kalyan Prasad
+- Kumar Anirudha
+- Laisha Wadhwa
+- Megha
+- Minal
+- Nabarun Pal
+- Nikhil Maan
+- Pradhvan Bisht
+- Pravin Patil
```
Comment (bhavin192) [inline content/team.md:37]:
```suggestion
- Praveen Patil
```

File: content/team.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- Garvit Kulshreshtha
+- Girish
+- Hari priya
+- Harsh Mittal
+- Jatin Goel
+- Kalyan Prasad
+- Kumar Anirudha
+- Laisha Wadhwa
+- Megha
+- Minal
```
Comment (bhavin192) [inline content/team.md:33]:
```suggestion
- Minal Patil
```

File: content/team.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+- Nabarun Pal
+- Nikhil Maan
+- Pradhvan Bisht
+- Pravin Patil
+- Priyanshu Kumar
+- Rahul B
+- Rahul Kumaresan
+- Rayan Das
+- Ritesh
```
Comment (bhavin192) [inline content/team.md:42]:
```suggestion
- Ritesh Agrawal
```

File: content/team.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+- Rohan Samanta
+- Sanchit
```
Comment (bhavin192) [inline content/team.md:44]:
```suggestion
- Sanchit Balchandani
```

File: content/team.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+- Sandeep Choudhary
+- Saptak
```
Comment (bhavin192) [inline content/team.md:46]:
```suggestion
- Saptak Sengupta
```

File: content/team.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+- Sayan
```
Comment (bhavin192) [inline content/team.md:47]:
```suggestion
- Sayan Chowdhury
```

File: content/team.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+- Shashank Kumar
+- Shivang Dubey
+- Subho
+- Suchita Pallavi
+- Sukanya Mandal
+- Tiyasha Banerjee
+- Vaibhav Mishra
+- Vipul Gupta
+- Vishal
+- Vishal VVR
```
Comment (bhavin192) [inline content/team.md:57]:
```suggestion
- Vishal Vijayraghavan
```

File: content/team.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+- Sayan
+- Shashank Kumar
+- Shivang Dubey
+- Subho
+- Suchita Pallavi
+- Sukanya Mandal
+- Tiyasha Banerjee
+- Vaibhav Mishra
+- Vipul Gupta
+- Vishal
```
Comment (bhavin192) [inline content/team.md:56]: @realslimshanky did we have another Vishal than Vishal Vijayraghavan (who handled stage 4)?

Probably these two are just same.

File: content/team.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+- Vishal VVR
+- Yashwanth Ravula
+
+This year, we had Chandan Kumar as the chair of the conference with Sayan Choudhary as the emertius chair. We also had 2 co-chairs, Kumar Aniruddha and Ritesh ?. If you too are interested in volunteering for next iteration of PyCon India, then [subscribe to the mailing list](https://mail.python.org/mailman/listinfo/inpycon) and wait for the call for volunteers for 2022 to start.
```
Comment (bhavin192) [inline content/team.md:60]:
```suggestion
This year, we had Chandan Kumar as the chair of the conference with Sayan Chowdhury as the emertius chair. We also had 2 co-chairs, Kumar Anirudha and Ritesh Agrawal. If you too are interested in volunteering for next iteration of PyCon India, then [subscribe to the mailing list](https://mail.python.org/mailman/listinfo/inpycon) and wait for the call for volunteers for 2022 to start. 
```

**Review by bhavin192 — CHANGES_REQUESTED** [review]

File: content/team.md (hunk 9 above)
Comment (SaptakS) [reply content/team.md:46]: `Saptak S` not `Saptak Sengupta`

File: content/team.md (hunk 12 above)
Comment (realslimshanky) [reply content/team.md:58]: @bhavin192 they are same.

**Review by realslimshanky — APPROVED** [review]

File: content/team.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+- Sayan Chowdhury
+- Shashank Kumar
+- Shivang Dubey
+- Subho
+- Suchita Pallavi
+- Sukanya Mandal
+- Tiyasha Banerjee
+- Vaibhav Mishra
+- Vipul Gupta
+- Vishal
```
Comment (bhavin192) [inline content/team.md:58]:
```suggestion
```

File: content/team.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+- Bhavin Gandhi
+- Chandan Kumar
+- Darshna Das
+- Divya Goswami
+- Duranjai Malik
+- Farhaan Bukhsh
+- Gajendra Deshpande
+- Garvit Kulshreshtha
+- Girish Joshi
+- Hari priya
```
Comment (bhavin192) [inline content/team.md:28]: Not sure if this is correct name, her name on Zulip is Haripriya.

**Comment by raukadah** [comment]
@vipulgupta2048 Can we add all working group leads to this review and verify all the volunteers ? Then I can give the final approval.

File: content/team.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+- Harsh Mittal
+- Jatin Goel
+- Kalyan Prasad
+- Kumar Anirudha
+- Laisha Wadhwa
+- Megha
+- Minal Patil
```
Comment (sukanyamandal) [inline content/team.md:35]: Manaswini Das's name is missing. She helped me throughout stage 5.

**Review by raukadah — APPROVED** [review]
Thanks Vipul & Team for working on it 😊😇😇

File: content/team.md [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+- Rayan Das
+- Ritesh Agrawal
+- Rohan Samanta
+- Sanchit Balchandani
+- Sandeep Choudhary
+- Saptak S
+- Sayan Chowdhury
+- Shashank Kumar
+- Shivang Dubey
+- Subho
```
Comment (rayandas) [inline content/team.md:53]: Can you please make it Subhendu Ghosh? It's his full name.

File: content/team.md [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+- Praveen Patil
+- Priyanshu Kumar
+- Rahul B
+- Rahul Kumaresan
+- Rayan Das
+- Ritesh Agrawal
+- Rohan Samanta
+- Sanchit Balchandani
+- Sandeep Choudhary
+- Saptak S
```
Comment (rayandas) [inline content/team.md:49]: Also Saptak Sengupta if we're using full name of everyone.

Comment (bhavin192) [reply content/team.md:49]: Saptak has asked to keep it Saptak S

**Comment by rayandas** [comment]
@vipulgupta2048 Also please add Siddhika Rakshe, she was active on promotion wg initially.

**Comment by abhishekmishragithub** [comment]
> @vipulgupta2048 Also please add Siddhika Rakshe, she was active on promotion wg initially.

Supratim as well.

**Comment by kalyan678** [comment]
I see the a wrong name in the list. It is "Priyanshu Krishnan" instead of "Priyanshu Kumar"

File: content/team.md [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+- Praveen Patil
+- Priyanshu Krishnan
+- Rahul B
+- Rahul Kumaresan
+- Rayan Das
+- Ritesh Agrawal
+- Rohan Samanta
+- Sanchit Balchandani
+- Sandeep Choudhary
+- Saptak Sengupta
```
Comment (bhavin192) [inline content/team.md:49]: https://github.com/pythonindia/inpycon2021/pull/64#discussion_r712787628

**Review by raukadah — APPROVED** [review]

**Pull request merged**